apps/dapp/src/components/Footer/Footer.tsx

```diff
@@ -20,80 +20,34 @@ export const Footer = () => {
         <div>
           <h3 className={'margin-remove'}>TempleDAO</h3>
           <br />
-          <p>
-            In a world of volatility and stress, the Temple of long term
-            wealth creation is where disciples come to stake, sleep easy,
-            and chill.
-          </p>
-          <br />
-          <br />
-          <strong>
-            &copy; {new Date().getFullYear()} TempleDAO. All rights
-            reserved.
-          </strong>
+          <CopyrightStyled>&copy; {new Date().getFullYear()} TempleDAO. All rights reserved.</CopyrightStyled>
         </div>
         <FooterGrid>
           <Column>
             <h4>Community</h4>
             <ul>
               <li>
                 {/* TODO: add no refere for SEO  */}
```

Comment:
Unrelated to your changes but do you know what this comment is about?

Reply:
I am not sure, no sorry.